src/SparkFun_u-blox_SARA-R5_Arduino_Library.cpp

```diff
@@ -246,6 +246,8 @@ bool SARA_R5::bufferedPoll(void)
           c = '0'; // Convert any NULLs to ASCII Zeros
         _saraRXBuffer[avail++] = c;
         timeIn = millis();
+      } else {
+        yield();
```

Comment:
Likewise for yield. I will need to check that this compiles on non-ESP32, non-threaded platforms.

Reply:
This is OK too. Even the ancient ATmega core includes a hook for yield...